src/os/CMakeLists.txt

```diff
@@ -92,6 +98,10 @@ if(HAVE_LIBAIO)
   target_link_libraries(os ${AIO_LIBRARIES})
 endif(HAVE_LIBAIO)
 
+if(HAVE_LIBZBC)
+  target_link_libraries(os ${ZBC_LIBRARIES})
+endif(HAVE_LIBZBC)
```

Comment:
could you remove the `HAVE_LIBZBC` in `endif()`. i understand that you are following the examples above. we will need to clean them up in future, though.

Reply:
Done.